src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -138,7 +138,7 @@ class PackageDetailTab extends mixin(StoreMixin) {
     const { params, location } = this.props;
 
     router.push(
-      `/catalog/packages/${encodeURIComponent(params.packageName)}/deploy?${location.query.version}`
+      `/catalog/packages/${encodeURIComponent(params.packageName)}/deploy?version=${location.query.version}`
```

Comment:
Haha pretty small change but to get here took me 1h to remember/read the code and see how we handle that again.

always fun to debug though.